geomesa-fs/geomesa-fs-tools/src/main/scala/org/locationtech/geomesa/fs/tools/ingest/FileSystemConverterJob.scala

```diff
@@ -44,7 +44,8 @@ trait FileSystemConverterJob extends StorageConfiguration with JobWithLibJars wi
           reducers: Int,
           libjarsFile: String,
           libjarsPaths: Iterator[() => Seq[File]],
-          statusCallback: StatusCallback): (Long, Long) = {
+          statusCallback: Option[StatusCallback],
```

Comment:
I think we've got two conflicting things going on here - as it is, if you set `waitForCompletion` to false but provide a `statusCallback`, it will still wait. I'd revert back to `statusCallback` not being an `Option`